extensions/amp-story/1.0/amp-story.js

```diff
@@ -690,12 +690,10 @@ export class AmpStory extends AMP.BaseElement {
       return;
     }
 
-    this.paginationButtons_ = PaginationButtons.create(this.win);
+    this.paginationButtons_ =
+        PaginationButtons.create(this.win, () => this.hasBookend_());
```

Comment:
This means that we're going to prefetch the bookend on page load on every desktop (and tablets?) device. It's not blocking, but we don't need it right now.
What would it take to do this refactoring without always prefetching the bookend?